src/applications/registry.json

```diff
@@ -262,12 +262,22 @@
     }
   },
   {
-    "appName": "New 686",
+    "appName": "686c-674",
     "entryName": "new-686",
```

Comment:
I wonder if `entryName` should change too. Might be worth changing this to mirror the appName.

Reply:
Changed this